CMakeLists.txt

```diff
@@ -352,6 +352,10 @@ enable_testing()
 #-----------------------------------------------------------------------------#
 # Check options, find packages and configure build.
 
+# required for time limits on non-POSIX systems
+find_package(Threads REQUIRED)
```

Comment:
If this is only required for the Windows cross-compilation we should guard this and not require it for all builds.

Reply:
I agree. I've added a check for the availability of `setitimer` and moved this into `ConfigureCVC4.cmake`